src/test/resources/testcontainers.properties

```diff
@@ -0,0 +1,2 @@
+# Digest of image testcontainersofficial/ryuk@sha256 for linux/amd64 from tags list on https://hub.docker.com/r/testcontainersofficial/ryuk
```

Comment:
Presumably this should identify the version e.g. “Digest of image testcontainersofficial/ryuk:latest …” or “Digest of image testcontainersofficial/ryuk:0.3.0 …”

Reply:
Updated to `:latest`